TravelerApp/TravelerAppCore/Controller/Menu.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using TravelerAppCore.Models.Hotels;
+
+namespace TravelerAppConsole
+{
+    public class Option
+    {
+
+
+        private static int num = 0;
+        public int _num { get; set; }
+        public string _text { get; set; }
+        public Del _met { get; set; }
```

Comment:
uważasz ze nazwy _met, Del, M1, n są nazwami dzięki którym jak czytasz kod to od razu wesz co oznaczają?

Reply:
dlaczego publiczne właściwości mają nazwę zaczynającą się od '_'

Reply:
Racja, powinienem na to zwrócić uwagę i to poprawić. Też informacja dla Joanny bo tutaj widać jej kawałek kodu.

Reply:
Poprawiono